wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneMigrator.swift

```diff
@@ -171,6 +172,10 @@ public struct OneOnOneMigrator: OneOnOneMigratorInterface {
                 mlsConversation.mutableMessages.union(proteusConversation.allMessages)
             }
 
+            // insert system message that we moved from proteus to MLS
+            let sender = ZMUser.selfUser(in: context)
+            mlsConversation.appendMLSMigrationFinalizedSystemMessage(sender: sender, at: .now)
+
```

Comment:
we insert the system message here, note that the migrator could run multiple times, though I have hard time to see in practice if that's possible (once moved to mls the conversation stays in MLS).